src/main/java/com/exactprosystems/remotehand/utils/ScreenshotUtils.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactprosystems.remotehand.utils;
+
+import com.exactprosystems.remotehand.ScriptExecuteException;
+import com.exactprosystems.remotehand.web.WebConfiguration;
+import com.jhlabs.image.PosterizeFilter;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriverException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class ScreenshotUtils {
+
+	private static final Logger logger = LoggerFactory.getLogger(ScreenshotUtils.class);
+
+	protected static final String SCREENSHOT_EXTENSION = ".png";
+	protected static final DateTimeFormatter SCREENSHOT_TIMESTAMP_FORMAT = DateTimeFormatter.ofPattern("yyyyMMddHHmmssSSS");
+	
+
+	public static String takeScreenshot(String name, TakesScreenshot takesScreenshot) throws ScriptExecuteException {
```

Comment:
Probably, it is better to call method `saveScreenshot` instead of `takeScreenshot` as this is what it does.
Similar method takeElementScreenshot doesn't save screenshot to disk and return its bytes. That' look confusing to me.